public/js/scripts.js

```diff
@@ -576,6 +606,9 @@ function submitQuestion() {
     });
 }
 
+function sendReply(id) {
```

Comment:
I think we don't need this anymore, It has been replaced right below by readyReply and Submit Reply. Probably is here because of an uncaught merge conflict.